docs/customize/ror_affiliations.md

```diff
@@ -0,0 +1,34 @@
+# ROR Affiliations
```

Comment:
I think this should be done via the ``invenio-cli services setup`` so that we avoid that the small ROR list is loaded.

Also, some information about how to check the progress. Perhaps we need another section (refereable to also from the upgrade guide on how to check the queue).

Reply:
Let's discuss before fixing

Reply:
Let's discuss IRL, the main problem with `setup` is that it will try to recreate the whole db/indices so it would only work for new instances not for existing/migrated ones.